### PR TITLE
add detailed description in README

### PR DESCRIPTION
**Pull request by serenejiang**
Add dependencies, description of running sequence and corresponding figures in manuscript

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,174 @@
...
+Figure 9b: boxplot_delivery_PC1_scores.pdf
+
+Figure 9c: boxplot_delivery_PC2_scores.pdf
+
+Figure 9d: boxplot_diet_PC1_scores.pdf
+
+Figure 9e: boxplot_diet_PC2_scores.pdf
+
+Figure 9f: boxplot_diet_PC3_scores.pdf
+
```
Comment (ElDeveloper) [inline README.md:166]: Can you remove these last few empty lines?

**Review by ElDeveloper — CHANGES_REQUESTED** [review]
@serenejiang, this looks good. My only suggestion would be to re-format the lists to be proper two-level bullet lists, for example like:

```
* observed data (sfpca_dfm.R)
  - Observed_spaghetti_group.pdf as Figure 1a; 
  - Observed_patient3.pdf as Figure 2a; 
  - Observed_patient5.pdf as Figure 2b
```

* observed data (sfpca_dfm.R)
  - Observed_spaghetti_group.pdf as Figure 1a; 
  - Observed_patient3.pdf as Figure 2a; 
  - Observed_patient5.pdf as Figure 2b

File: README.md (hunk 1 above)
Comment (serenejiang) [reply README.md:166]: Just did. Thanks!

**Comment by ElDeveloper** [comment]
Wonderful, looks great. Thanks!

**Pull request merged**